frontend/packages/console-app/src/components/cloud-shell/cloud-shell-utils.ts

```diff
@@ -35,22 +23,22 @@ export type CloudShellResource = K8sResourceKind & {
   };
   spec?: {
     started?: boolean;
-    devfile?: Devfile;
     routingClass?: string;
+    template?: DevWorkspaceTemplateSpec;
   };
 };
 
 export type TerminalInitData = { pod: string; container: string; cmd: string[] };
 
 export const CLOUD_SHELL_LABEL = 'console.openshift.io/terminal';
-export const CLOUD_SHELL_CREATOR_LABEL = 'org.eclipse.che.workspace/creator';
-export const CLOUD_SHELL_IMMUTABLE_ANNOTATION = 'org.eclipse.che.workspace/immutable';
+export const CLOUD_SHELL_CREATOR_LABEL = 'controller.devfile.io/creator';
+export const CLOUD_SHELL_IMMUTABLE_ANNOTATION = 'controller.devfile.io/restricted-access';
```

Comment:
nit: Should it be better to name it CLOUD_SHELL_RESTRICTED_ANNOTATION ? 
There are 2 other files to be changed in that case.

Reply:
While I'm renaming the constants, should I name them `WEB_TERMINAL_*` as well, given the naming change? (`cloud-shell -> web-terminal`)

Reply:
@amisevsk i'd keep it as cloud shell right now for consistency in the code as the entire feature is using cloud shell.
We don't surface `cloud shell` to the user.

Reply:
Renamed to `CLOUD_SHELL_RESTRICTED_ANNOTATION`